pyspectre/core.py

```diff
@@ -55,18 +57,19 @@ def read_results(raw_file: str, offset: int = 0) -> dict[str, DataFrame]:
 
     return plot_dict(read_raw(raw_file, off_set = offset))
 
-def simulate( netlist_path: str, includes: Iterable[str] = None
+def simulate( netlist_path: str, includes: List[str] = None
             , raw_path: str = None, log_path: str = None
-            ) -> dict[str, DataFrame]:
+            ) -> Dict[str, DataFrame]:
     """
     Passes the given netlist path to spectre and reads the results in.
     """
     net = os.path.expanduser(netlist_path)
     inc = [f'-I{os.path.expanduser(i)}' for i in includes] if includes else []
     raw = raw_path or raw_tmp(net)
-    log = ['-log']if not log_path else [f'=log {log_path}']
-    cmd = ['spectre', '-64', '-format nutbin', f'-raw {raw}'
-          ] + log + inc + [net]
+    log = f'{net}.log' if not log_path else f'{log_path}'
+    #log_path = log
+    # These individual arguments cannot be combined with the argument options, those are not recognized
+    cmd = ['spectre', '-64', '-format', 'nutbin', '-raw', f'{raw}', '+log', f'{log}'] + inc + [net]
```

Comment:
This hardcodes `+log` which I don't like at all. In fact that might break something on my end. We need a way to disable the log and we do so in serafin. Writing the log to file will cause significant slowdown for large simulations, as the file grows. Additionally Printing the log to the REPL causes slowdown and makes it harder to parse.

Reply:
Yes, I needed this to debug, but once we know what works, we should enable logging conditionally, but except for specific cases run log, with what was run and other aspects is not supposed to be huge.

Reply:
We are using this as backend for a RL environment. If your running over +10k simulations the log files can get really large, several GBs in fact. Since new logs are always appended to the same file this operations slows down everything. It's actually a really delicate thing.